docs/reference/standards/guides/issuing-api-calls.md

```diff
@@ -1,5 +1,9 @@
 ---
```

Comment:
As I commented in previous PR https://github.com/ava-labs/avalanche-docs/pull/1460/files/68d321039c286990738dc6c4aec3fe77b0f6b7a2#r1269523596, this should be moved. If not, it should be referenced to in each of the API doc so that users would know how to make the call.

Reply:
Going to reiterate that making API calls is not an AvalancheGo specific functionality, and is consistent for most public RPCs. This doc exists to supplement the tutorials that include making API calls to avalanchego but does not need to be front and center on this technical reference tab. The doc will be prioritized in the sidebar of the page covering DApp and Node development.

Reply:
I agree with Meaghan -- I think this is a fine place for this information.